securitycenter/findingsv2/findings_test.go

```diff
@@ -137,45 +153,67 @@ func TestCreateFinding(t *testing.T) {
 }
 
 func TestUpdateFindingSourceProperties(t *testing.T) {
-	setup(t)
-	testutil.Retry(t, 5, 5*time.Second, func(r *testutil.R) {
-		buf := new(bytes.Buffer)
-
-		err := updateFindingSourceProperties(buf, findingName)
-
-		if err != nil {
-			r.Errorf("updateFindingSourceProperties(%s) had error: %v", findingName, err)
-			return
-		}
-
-		got := buf.String()
-		if want := "s_value"; !strings.Contains(got, want) {
-			r.Errorf("updateFindingSourceProperties(%s) got: %s want %s", findingName, got, want)
-		}
-		if !strings.Contains(got, findingName) {
-			r.Errorf("updateFindingSourceProperties(%s) got: %s want %s", findingName, got, findingName)
-		}
-	})
+    setup(t)
+    testutil.Retry(t, 5, 5*time.Second, func(r *testutil.R) {
+        ctx := context.Background()
+        client, err := securitycenter.NewClient(ctx)
+        if err != nil {
+            r.Errorf("Failed to create client: %v", err)
+        }
+        defer client.Close()
+
+        finding, err := createTestFinding(ctx, client, "update-finding", "MEDIUM_RISK_ONE")
```

Comment:
use [`t.Cleanup()`](https://pkg.go.dev/testing#T.Cleanup) to delete these test findings, so they do not get left behind after tests complete.

Reply:
Addressed